src/components/ControlPanel/ControlPanel.js

```diff
@@ -25,16 +44,36 @@ export const ControlPanel = InjectAppServices(({ dependencies: { controlPanelSer
           {controlPanelSections.map((section, indexSection) => (
             <div key={`section-${indexSection}`} className="col-lg-12 col-md-12 m-b-24">
               <div className="dp-bg-ghostwhite dp-box-shadow m-b-24">
-                <h2>{_(section.title)}</h2>
+                <S.TitleContainer>
+                  <h2>{_(section.title)}</h2>
+                  {section.showStatus ? (
+                    <S.StatusBoxContainer>
+                      <S.StatusIcon src={connection_alert} alt="connection alert icon" />
+                      <span className="yellow-color">{_('control_panel.status_alert')}</span>
+                      <S.StatusIcon src={disconnected} alt="disconnected icon" />
+                      <span className="lightgrey-color">
+                        {_('control_panel.status_not_connected')}
+                      </span>
+                      <S.StatusIcon src={connected} alt="connection icon" />
+                      <span className="lightgreen-color">
+                        {_('control_panel.status_connected')}
+                      </span>
+                    </S.StatusBoxContainer>
+                  ) : (
+                    <></>
+                  )}
+                </S.TitleContainer>
                 <div className="dp-rowflex">
-                  {section.boxes.map((box, indexBox) => (
-                    <ControlPanelBox
-                      box={box}
-                      key={`box-${indexBox}`}
-                      disabled={!!box.disabled}
-                      hidden={!!box.hidden}
-                    />
-                  ))}
+                  {controlPanelSections[indexSection].boxes
+                    .sort(sortByStatus)
+                    .map((box, indexBox) => (
+                      <ControlPanelBox
+                        box={box}
+                        key={`box-${indexBox}`}
+                        disabled={!!box.disabled}
+                        hidden={!!box.hidden}
+                      />
```

Comment:
I understand that it is out of the scope of the PR, but maybe for a future PR, what do you think about that simply read `disabled` and `hidden` properties inside `ControlPanelBox`?:

```suggestion
                      <ControlPanelBox
                        box={box}
                        key={`box-${indexBox}`}
                      />
```

Reply:
I did it like that to explicitly set a default value for those properties. But I guess it's not much of a difference. I will remove it in another PR 😉